src/pages/projectCreate/hooks/useWalletForm.tsx

```diff
@@ -178,11 +184,12 @@ export const useWalletForm = ({
           lightningAddress: lightningAddressFormValue,
         },
         resourceInput,
+        feePercentage,
```

Comment:
Do we have the feePercentage, for lightning addresses set as well? It might be better to remove this right now.

Reply:
The `feePercentage` in the backend is in the `wallet` table that holds both lnd and lightning. It is set to 0 when it's a lightning wallet that is being used. I could:
1. send a fee percentage of 0 (using the enum) when it's a lightning address 
2. or make it options in the walletform type and just avoid sending it altogether